examples/plot_minimal_pydeseq2_pipeline.py

```diff
@@ -0,0 +1,130 @@
+"""
+PyDESeq 2 pipeline
+===========================
+
+This notebook gives a minimalistic example of how to perform DEA using PyDESeq2.
+
+It allows you to run the PyDESeq2 pipeline on synthetic data provided as part of this repository.
+"""
+
+import os
+import pickle as pkl
+
+from pydeseq2.DeseqDataSet import DeseqDataSet
+from pydeseq2.DeseqStats import DeseqStats
+from pydeseq2.utils import load_example_data
+
+SAVE = False  # whether to save the outputs of this notebook
```

Comment:
If we keep the `SAVE` flag we should also keep `OUTPUT_PATH`:

```suggestion
SAVE = False  # whether to save the outputs of this notebook

if SAVE:
    OUTPUT_PATH = f"../output_files/synthetic_example"
    os.makedirs(OUTPUT_PATH, exist_ok=True)  # Create path if it doesn't exist
```

(Should we adapt the path stored as `OUTPUT_PATH`, or keep it as is?)